packages/backend/src/routers/dashboardRouter.ts

```diff
@@ -47,6 +47,26 @@ dashboardRouter.patch(
     },
 );
 
+dashboardRouter.patch(
+    '/pinning',
```

Comment:
should it be `pin` ?

Reply:
Since it is working as a toggle we named it like that. Would the FE call the `pin` endpoint to unpin the item? 
Let me know what you think